client/src/components/useraccess/Forms/PasswordResetModal.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import * as userActions from '../../../Actions/userAccessAction';
+
+/**
+ * @class PasswordResetModal
+ * @classdesc returns Password reset modal
+ */
+class PasswordResetModal extends React.Component {
+  constructor(prop) {
+    super();
+    this.state = {
+      email: ''
+    }
+
+    this.handleInput = this.handleInput.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.cancelPassReset = this.cancelPassReset.bind(this);
+    const message = document.getElementById('message');
+  }
+
+  /**
+   * 
+   * @param { object } event 
+   * @memberof PasswordResetModal
+   * @returns { object } a new state with use email
+   */
+  handleInput(event) {
+    this.setState({email: event.target.value})
+  }
+
+  /**
+   * 
+   * 
+   * @param { object } event 
+   * @memberof PasswordResetModal
+   * @returns { object } respose from api call 
+   */
+  handleSubmit(event) {
+    // validate input
+    if(this.state.email === "") {
+    message.innerHTML = `Your email is needed to reset your password`;
+    message.className="red-text center-align";  
+    } else {
+      
+      //Make API call
+      this.props.sendEmail({email: this.state.email})
+      .then(() => {
+        message.innerHTML = `A password reset link has been sent to ${this.state.email}`;
+        message.className="green-text center-align";
+      })
+      .catch((error) => {
+        if (error.response.status === 501) {
+          message.innerHTML ="Ooops! Something went wrong, the server could not process your request at this time. Please try again.";
+          message.className="red-text center-align";
+        } else {
+          message.innerHTML = error.response.data.message;
+          message.className="red-text center-align";
+        }
+      })
+    }
+    // prevent button default action
+    event.preventDefault();
```

Comment:
event.preventDefault should be the first method called when this method is called.

Reply:
Alright. Noted and will be fixed.